middleware/privilegetoken/claims.go

```diff
@@ -0,0 +1,42 @@
+package privilegetoken
+
+import (
+	"encoding/json"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/golang-jwt/jwt/v4"
+)
+
+type TokenClaims struct {
+	DeviceTokenID  string // Entity in this case is what we need privilege to access
+	UserEthAddress string
+	PrivilegeIDs   []int64
+}
+
+type privilegeTokenClaimsResponseRaw struct {
+	Sub            string
+	UserEthAddress string
+	PrivilegeIDs   []int64
```

Comment:
I think you're calling this `"privileges"` still in the token. That is fine, but one of these has to change.